src/locales/fr/texts.json

```diff
@@ -0,0 +1,277 @@
+  },
+  "description": "Description",
+  "errors": {
+    "authFailed": "La requête a échoué, les informations d'authentification fournies peuvent être incorrectes.",
+    "badRequest": "La requête n'est pas valide. Cela peut être dû à des paramètres de recherche invalides ou à un bogue au niveau de STAC Browser",
+    "contactProvider": "Veuillez contacter l'exploitant du service afin de résoudre ce problème.",
+    "default": "La page demandée n'a pas pu être chargée.",
+    "forbidden": "Les informations d'authentification spécifiées pour cette requête ne sont pas valides, par exemple un jeton d'API expiré ou est non valide. Veuillez fournir d'autres informations d'authentification et réessayer",
+    "invalidJsonObject": "La réponse n'est pas un objet JSON valide.",
+    "invalidStacCollections": "La réponse de l'API n'est pas une liste valide de collections STAC.",
+    "invalidStacItems": "La réponse de l'API n'est pas une liste valide d'éléments STAC.",
+    "labels": {
+      "cause": "Cause:",
+      "code": "Code d'erreur:",
+      "reason": "Raison:",
+      "requestURL": "URL demandé:"
+    },
+    "loadApiCollectionsFailed": "Les collections de l'API n'ont pas pu être chargées.",
+    "loadApiItemsFailed": "Les éléments de l'API n'ont pas pu être chargés.",
+    "loadFilteredItems": "Désolé, le chargement d'une liste d'éléments filtrés a échoué.",
+    "loadItems": "Désolé, le chargement de la liste des éléments STAC a échoué.",
+    "networkError": "Ce problème peut se produire lorsque les serveurs n'autorisent pas l'accès externe via les navigateurs Web (par exemple, lorsque les en-têtes CORS ne sont pas présents). @:browse.contactProvider",
+    "noExternalAccess": "L'accès aux catalogues externes n'est pas autorisé!",
+    "notFound": "La ressource demandée n'existe pas. @:browse.contactProvider",
+    "serverError": "Le serveur a rencontré un problème. @:browse.contactProvider",
+    "unauthorized": "La requête requiert des informations d'authentification, comme par exemple un jeton API. Veuillez fournir vos informations d'authentification et réessayer"
+  },
+  "featureExperimental": "Cette fonctionnalité est encore expérimentale et peut donner des résultats inattendus!",
+  "fields": {
+  },
+  "fullscreen": {
+    "exit": "Quitter le mode plein écran",
+    "show": "Voir en plein écran"
+  },
+  "goToCollection": {
+    "description": "Accéder à la collection associée",
+    "descriptionWithTitle": "@:goToCollection.description: {title}",
+    "label": "Collection"
+  },
+  "goToParent": {
+    "description": "Accéder au parent",
+    "descriptionWithTitle": "@:goToParent.description: {title}",
+    "label": "En haut"
+  },
+  "index": {
+    "api": "API",
+    "catalog": "Catalogue",
+    "load": "Charger",
+    "selectStacIndex": "... ou en sélectionner un dans l'index STAC:",
```

Comment:
"dans STAC Index"? STAC Index is a name so I think it shouldn't be translated.

Reply:
I believe STAC acts as a qualifier of the type of index, then 'index STAC' is more readable (noun + qualifier) and similar to 'la collection STAC' and 'le catalogue STAC' used elsewhere.